### PR TITLE
Add --skip-debezium flag in installer script to avoid installing debezium when only offline setup is required

### PR DESCRIPTION
**Pull request by sanyamsinghal**
### Describe the changes in this pull request
Added a new flag to installer script `--skip-debezium` to avoid installing debezium for cases like when only offline setup is required

```
./installer_scripts/install-yb-voyager  --install-from-local-source --skip-debezium
```

### Describe if there are any user-facing changes

If user doesn't provide this new flag(old way of invoking installler script) there won't be any change in behaviour for them.

### How was this pull request tested?

Existing tests ensure that nothing is getting affected with this new flag, unless its specified

### Does your PR have changes that can cause upgrade issues? 
| Component        | Breaking changes? |
| :----------------------------------------------: | :-----------: |
| MetaDB                                                    | No |
| Name registry json                                        |  No |
| Data File Descriptor Json                                 |  No |
| Export Snapshot Status Json                               | No |
| Import Data State                                         | No |
| Export Status Json                                        | No |
| Data .sql files of tables                                 | No |
| Export and import data queue                              |  No |
| Schema Dump                                               | No |
| AssessmentDB                                              | No |
| Sizing DB                                                 | No |
| Migration Assessment Report Json                          |  No |
| Callhome Json                                             | No |
| YugabyteD Tables                                          | No |
| TargetDB Metadata Tables                                  | No |

### CODE REVIEW

File: installer_scripts/install-yb-voyager [hunk 1]
```diff
@@ -555,6 +560,10 @@ get_passed_options() {
 				ONLY_PG="true"; 
 				shift 
 				;;
+			-n | --no-debezium )
```
Comment (makalaaneesh) [inline installer_scripts/install-yb-voyager:563]: This is a subjective thing but how about "skip-debezium" or "exclude-debezium" instead? "no-" prefix feels slightly weird imo in the context of installation.   
Also `-n` is a bit confusing. (Same with `-v` that we currently have as well... ). What if we want to add "no-ora2pg" or "skip-ora2pg" in the future?
Can you explore if we can have two-letter short-forms?

Comment (sanyamsinghal) [reply installer_scripts/install-yb-voyager:563]: `skip-debezium` sounds good. let me explore `two-letter short-forms`

Comment (sanyamsinghal) [reply installer_scripts/install-yb-voyager:563]: Doesn't look like that is possible.
```
For example, -ab is interpreted as two short options: -a and -b.
```

But i think what we can do is just have long forms for some of the options.

Comment (makalaaneesh) [reply installer_scripts/install-yb-voyager:563]: I think on mac, long-form options don't work. Let's just choose some letter then in that case. maybe "-d"  for skip-debeizum?

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/4299/

File: installer_scripts/install-yb-voyager [hunk 2]
```diff
@@ -555,7 +560,11 @@ get_passed_options() {
 				ONLY_PG="true"; 
 				shift 
 				;;
-			-v | --rebuild-voyager-local )
+			--skip-debezium )
```
Comment (makalaaneesh) [inline installer_scripts/install-yb-voyager:563]: let's add -v back and -d for skip-debezium.

File: installer_scripts/install-yb-voyager [hunk 3]
```diff
@@ -538,7 +543,7 @@ rebuild_voyager_local() {
 get_passed_options() {
 	if [ "$1" == "linux" ]
 	then
-		OPTS=$(getopt -o "lpvV", --long install-from-local-source,only-pg-support,rebuild-voyager-local,version: --name 'install-yb-voyager' -- $ARGS_LINUX)
+		OPTS=$(getopt -o "lpnvV", --long install-from-local-source,only-pg-support,skip-debezium,rebuild-voyager-local,version: --name 'install-yb-voyager' -- $ARGS_LINUX)
```
Comment (makalaaneesh) [inline installer_scripts/install-yb-voyager:546]: lpvsV?

**Review by makalaaneesh — APPROVED** [review]
LGTM with couple of comments

File: installer_scripts/install-yb-voyager (hunk 3 above)
Comment (sanyamsinghal) [reply installer_scripts/install-yb-voyager:546]: yes, `lpdvV` where -d is for skip debezium

**Pull request merged**